openapi/publicapi_spec.json

```diff
@@ -1211,7 +1211,7 @@
           "created_date" : {
             "type" : "string",
             "description" : "The date you created the payment.",
-            "example" : "2016-01-21T17:15:00Z"
+            "example" : "2016-01-21T17:15:00.000Z"
```

Comment:
This change looks correct (we always provide milliseconds in our API responses). Perhaps the millisecond number should be `123` or similar though (it’s unlikely it would actually be `000`).